src/extensions/UserDefinedFormExtension.php

```diff
@@ -7,22 +7,29 @@
 use SilverStripe\Forms\FieldList;
 use SilverStripe\Forms\GridField\GridField;
 use SilverStripe\Forms\GridField\GridFieldAddNewButton;
-use SilverStripe\Forms\GridField\GridFieldConfig_RelationEditor;
+use SilverStripe\Forms\GridField\GridFieldConfig_RecordEditor;
 use SilverStripe\Forms\Tab;
 use SilverStripe\ORM\DataExtension;
+use SilverStripe\ORM\DataList;
+use SilverStripe\UserForms\Model\UserDefinedForm;
 
 /**
- * Class \Firesphere\PartialUserforms\Extensions\UserDefinedFormExtension
+ * Class UserDefinedFormExtension
  *
+ * @package Firesphere\PartialUserforms\Extensions
```

Comment:
Did you type all these by hand? :O

Reply:
Nope, package is automatically added when I create a docblock 😸